dataBase/dummyData.js

```diff
@@ -0,0 +1,171 @@
+const data = [
```

Comment:
This is just a suggestion, but you can use javascript to dynamically render the dates for you using the date() method.